notebooks/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>The thing I'm most missing in this check's output is the "bottom line", which should appear first. </p><p>A clear indicator to the question "are they drifted"? and "how much"? (which is also the results on which the condition is applied, right?)</p><p> </p><p>The rest of the graphs are great to explain the drift, but that's interesting mainly if they are drifted...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
and please add conditions' demonstration to notebook

Reply:
Condition - will do
Bottom line - I can add a traffic light bar like in the regular drift as the first plot.

As for your other question on "what happens if no features pass" - then none are shown, but feature importance tends to be distributed non-evenly, so as long as your threshold is less then 0.1 my guess is that something will be shown.